boxoffice/mailclient.py

```diff
@@ -55,15 +55,15 @@ def send_line_item_cancellation_mail(line_item_id, subject="Ticket Cancellation"
 
 
 @job('boxoffice')
-def send_ticket_assignment_mail(line_item_id):
+def send_ticket_assignment_mail(line_item_id, recipient_list, cc_list):
```

Comment:
I think `cc_list` should have `[]` as its default parameter since it isn't mandatory.